src/components/AppNavi/AppNaviButton.tsx

```diff
@@ -74,8 +74,19 @@ const CurrentWrapper = styled.span<{ themes: Theme }>`
 
     return css`
       ${BaseStyle}
-      border-bottom: ${size.pxToRem(3)} solid ${palette.MAIN};
       color: ${palette.TEXT_BLACK};
+      position: relative;
+
+      &:after{
```

Comment:
[MUST] As uknmr says, `:` should be two. Also, put a space after `after`.
This is what linter can handle, but it's not ready yet, sorry for the trouble :pray: